raiden/network/transport/matrix/transport.py

```diff
@@ -427,16 +427,17 @@ def on_success(greenlet):
         # Handle any delayed invites in the future
         self._schedule_new_greenlet(self._process_queued_invites, in_seconds_from_now=1)
 
-    def _process_queued_invites(self):
+    def _process_queued_invites(self) -> None:
         if self._invite_queue:
             self.log.debug("Processing queued invites", queued_invites=len(self._invite_queue))
             for room_id, state in self._invite_queue:
                 self._handle_invite(room_id, state)
             self._invite_queue.clear()
 
-    def _run(self):
+    def _run(self) -> None:  # type: ignore
         """ Runnable main method, perform wait on long-running subtasks """
         # dispatch auth data on first scheduling after start
+        assert self._raiden_service is not None, "_raiden_service not set"
```

Comment:
This was the problem we encountered before, we have tu use `is not None` here, because `Runnable` overwrites `__bool__`

@ulope

Reply:
I also created https://github.com/raiden-network/raiden/pull/5090 now